hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,134 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def generate_outside(box, interior_points, multipliers, initial_images):
+    """Generate test cases from interior points by adding box vectors."""
+    # construct unit cell from box vectors
+    # see
+    # hoomd-blue.readthedocs.io/en/latest/package-hoomd.html#hoomd.Box.from_matrix # noqa
+    a = numpy.array([box[0], 0, 0])
+    b = numpy.array([box[1] * box[3], box[1], 0])
+    c = numpy.array([box[2] * box[4], box[2] * box[5], box[2]])
```

Comment:
I started out doing everything manually to avoid using code used in the wrap itself, but this is just at the python level so good to simplify.

`f` was a shorter form of `factor` to help avoid as much line-wrapping as possible